akka-docs/src/main/paradox/scala/io-udp.md

```diff
@@ -17,12 +17,16 @@ offered using distinct IO extensions described below.
 
 ### Simple Send
 
-@@snip [UdpDocSpec.scala]($code$/scala/docs/io/UdpDocSpec.scala) { #sender }
+Scala
+:  @@snip [UdpDocSpec.scala]($code$/scala/docs/io/UdpDocSpec.scala) { #sender }
+
+Java
+:  @@snip [UdpDocTest.java]($code$/java/jdocs/io/UdpDocTest.java) { #sender }
 
 The simplest form of UDP usage is to just send datagrams without the need of
 getting a reply. To this end a “simple sender” facility is provided as
 demonstrated above. The UDP extension is queried using the
-`SimpleSender` message, which is answered by a `SimpleSenderReady`
+@scala[`SimpleSender` message]@java[`UdpMessage.simpleSender` method], which is answered by a `SimpleSenderReady`
```

Comment:
In Java the extension is still queried with a message (not a method), but the message is created by calling the factory that is mentioned here in the docs.